build.html

```diff
@@ -3,4 +3,150 @@
+    </div>
+    <div class="ds-l-col--12 ds-u-text-align--right ds-u-font-size--h1 ds-u-color--white">
+      The User Guide for ACO's in Production
+    </div>
+  </div>
+
+  <div class="ds-l-row ds-u-padding-top--2">
+
+    <!-- Table of Contents -->
+    <aside class="ds-u-margin-bottom--4 ds-u-md-margin-bottom--0 ds-l-md-col--4 ds-l-lg-col--3">
+      <ul class="ds-c-vertical-nav">
+        <li class="ds-c-vertical-nav__item">
+          <a
+            class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+            href="#bcda-user-guide"
+          >
+            BCDA User Guide Documentation
+          </a>
+        </li>
+        <li class="ds-c-vertical-nav__item">
+          <a
+            class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+            href="#request-metadata"
+          >
+          Request Metadata
+          </a>
+        </li>
+        <li class="ds-c-vertical-nav__item">
+          <a
+            class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+            href="#authorization"
+          >
+            Authorization and Authentication
+          </a>
+          <ul class="ds-c-vertical-nav__subnav">
+            <li class="ds-c-vertical-nav__subnav">
+              <a
+                class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+                href="#bcda-credentials"
+              >
+                Obtain your BCDA Credentials
+              </a>
+            </li>
+            <li class="ds-c-vertical-nav__subnav">
+              <a
+                class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+                href="#access-token"
+              >
+                Obtain your Access Token
+              </a>
+            </li>
+          </ul>
+        </li>
+        <li class="ds-c-vertical-nav__item">
+          <a
+            class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+            href="#requesting-data"
+          >
+            Requesting Data
+          </a>
+          <ul class="ds-c-vertical-nav__subnav">
+            <li class="ds-c-vertical-nav__subnav">
+              <a
+                class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+                href="#request-all"
+              >
+                Request All Three Resource Types
+              </a>
+            </li>
+            <li class="ds-c-vertical-nav__subnav">
+              <a
+                class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+                href="#request-filtered"
+              >
+                Requesting Filtered Data
+              </a>
+            </li>
+            <li class="ds-c-vertical-nav__subnav">
+              <a
+                class="ds-c-vertical-nav__label ds-c-vertical-nav__label--current"
+                href="#request-recent-and-historical"
+              >
+                Request recent data AND historical data
```

Comment:
Do we need to add `Check the Job Status` and `Download the Data`?

Reply:
@mtrang1263 Consulting with Dave's updated site language, it appears that we include those as subheads under requesting data each time.